mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/overview/index.tsx

```diff
@@ -80,7 +80,8 @@ function getWooCommerceDelta(type: 'revenue' | 'orders'): number | undefined {
     return 0;
   }
 
-  return (newValue / previous) * 100;
+  const delta = (newValue / previous) * 100;
+  return current > previous ? delta : delta * -1;
```

Comment:
I guess the original `current > previous ? current - previous : previous - current` is not needed and then it's simply:

```tsx
const newValue = current - previous;
return (newValue / previous) * 100;
```